src/main/webapp/main/LandingPage.js

```diff
@@ -0,0 +1,191 @@
+import React, { Component } from 'react';
+import { hh, h, div, h3, a } from 'react-hyperscript-helpers';
+import { About } from '../components/About';
+import { TableComponent } from "../components/TableComponent";
+import { Issues, User } from "../util/ajax";
+import { parseDate } from "../util/TableUtil";
+import { Link } from 'react-router-dom';
+
+const columnsCopy = [{
```

Comment:
Why is this named `columnsCopy`? Isn't it just the columns that are being shown for each table?